README.md

```diff
@@ -14,6 +14,17 @@ to alert data, therefore safe to be accessed by wider audience.
 Alertmanager's API isn't stable yet and can change between releases.
 unsee currently supports Alertmanager `0.4` and `0.5`.
 
+## Security
+
+unsee process doesn't send any API request to the Alertmanager that could modify
+alerts or silence state. But it does provide a web interface that allows a user
```

Comment:
Instead of starting a second sentence I think it should be a continuation, i.e. "or silence state, but it does"